llama_stack/providers/tests/inference/test_text_inference.py

```diff
@@ -118,6 +118,7 @@ async def test_completion(self, inference_model, inference_stack):
             "remote::fireworks",
             "remote::nvidia",
             "remote::cerebras",
+            "remote::vllm",
```

Comment:
can you just remove this block now entirely? whichever provider fails the test better just show up as a Failure.